parmed/rdkit/rdkit.py

```diff
@@ -32,10 +32,22 @@ def load(rmol):
         return PDBFile.parse(fh)
 
     @staticmethod
-    def from_smiles(smiles):
+    def from_smiles(smiles, coordinates=True):
         """
         Load smiles string to :class:`Structure`
+
+        Parameters
+        ----------
+        smiles : str, smiles
+        coordinates : bool, default True
+            if True, use `rdkit.Chem.AllChem.EmbedMultipleConfs to assign coordinates
+            if False, coordinates for all atoms are all 0.0
```

Comment:
because rdkit save to `mol2` file in `StringIO`, then `ParmEd` read that.

I can assign `None`